functions/exercises.py

```diff
@@ -52,15 +47,18 @@ def get_sum_of_list(num_list):
 def get_prod_of_list(num_list, absolute=False):
     """
     Return the product of all numbers in num_list
-
     :param num_list: List of numbers
     :param absolute: Use absulute values
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    product =  1
+    if absolute :
+        for num in num_list:
+            product *= abs(num)
+    else:
+        for num in num_list:
+            product *= num
+    return product

```

Comment:
```suggestion
    for num in num_list:
        product *= num
    if absolute :
            product = abs(product)
    return product
```
reducing code duplication is a great way to keep your code simple